### ISSUE TITLE
NVDA doesn't read controls on notification message

### ISSUE THREAD

**Issue by cait1inhennessy** [Microsoft/vscode#57996]
This accessibility bug was filed on the Live Share team; however, it is really an editor issue. I've pasted the bug description below, editing it slightly to remove irrelevant Live Share-specific content.

Repro Steps:
1) Start NVDA
2) Show any notification with the vscode.window.showNotificationMessage API
3) Check whether the Screen Reader is reading the contents of the notification properly

Actual Result:
NVDA is reading the entire content of the notification when the focus is on first interactive element of the notification, but when the focus is shifted to other control, NVDA is not reading anything

NOTE: Same is the issue with all the notifications that pop-up in the application
[Similar issue is observed in Mac as well.]

Expected Result:
NVDA should read all the controls of the notification properly

User Impact: 
Screen reader users will not be able to access the controls present in the notification

MAS Reference: 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

OS: Windows 10 Enterprise 17134.228

**Comment by cait1inhennessy** [comment]
@bpasero Do you know what version of VSCode will use Electron 3.0.x?

**Comment by cait1inhennessy** [comment]
@bpasero OK, thanks for the info.
